### DISCUSSION TITLE
Not seeing Endpoint or Trace data

### DISCUSSION THREAD

**Discussion by ke5vmj** [SkyAPM/SkyAPM-php-sdk#494] [Q&A]
Hi,

I'm trying to setup PHP instrumentation but whenever I hit endpoints on my PHP application I do see that the service is recognized and the instance is recognized too but I do not see Endpoint data or Trace data or anything else. What else is needed? Do I have to add further configuration to enable that or should I expect this out of the box? Is it an OAP server version? I think my OAP version is 9.0 and the skywalking agent version was 4.2.0 or something like that. Should I try to upgrade the agent or downgrade OAP?

**Comment by heyanlong** [comment]
Please upgrade skywalking-php to 5.0.0
